travis/travis_install_nightly

```diff
@@ -138,6 +138,14 @@ else
     # Install websocket-client, required to run JS tests
     pip install websocket-client
 fi
+# The new syntax of JS tests requires the latest tools to run
+if [ ${VERSION%.*} -ge 13 ]; then
+    wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb # downloading updated chrome dev tools
+    apt update
+    apt install fonts-liberation  # chrome dev tools dependency
+    apt install libgbm1  # chrome dev tools dependency
```

Comment:
- Install both commands in one line
- Comment ir superfluous, we're installing chrome along with their dependencies